src/webpack.js

```diff
@@ -29,13 +32,131 @@ const parseEntrypoint = (entrypoint) => {
   };
 };
 
+/**
+ * Helper function to trim absolute file path by removing the `process.cwd()`
+ * prefix if file is nested under `process.cwd()`.
+ */
+function makeFilePathRelativeToCwd (file) {
+  const cwd = process.cwd();
+  return (file.startsWith(cwd)) ? '.' + file.substring(cwd.length) : file;
+}
+
+/**
+ * @param {String} outputDir the directory that contains output files
+ * @param {String[]} entryNames the entrypoint names from which output was produced
+ */
+async function zipOutputFiles (outputDir, entryNames) {
+  console.log('\nCreating zip file for each entrypoint...\n');
+
+  for (const entryName of entryNames) {
+    const dirname = path.dirname(entryName);
+    // Find all of the output files that belong to this entry
+    const entriesForZipFile = (await glob(`${entryName}*`, {
+      cwd: outputDir
+    })).map((file) => {
+      return {
+        name: (dirname === '.') ? file : file.substring(dirname.length + 1),
+        file: path.join(outputDir, file)
+      };
+    });
+
+    // Now, write a zip file for each entry
+    console.log(`Creating zip for entrypoint ${chalk.bold(entryName)}...`);
+    console.log(entriesForZipFile.map((entry) => {
+      return `- ${chalk.bold(entry.name)}\n`;
+    }).join(''));
+    const outputZipFile = path.join(outputDir, `${entryName}.zip`);
+    await zip(outputZipFile, entriesForZipFile);
+    console.log(chalk.green(`Zip file for ${chalk.bold(entryName)} written to ` +
+      `${chalk.bold(makeFilePathRelativeToCwd(outputZipFile))}\n`));
+  }
+}
+
+const INDEX_FILES = ['index.js', 'index.ts'];
+
+async function findIndexFile (dir) {
+  for (const indexFile of INDEX_FILES) {
+    const candidateFile = path.join(dir, indexFile);
+    try {
+      const stats = await fs.stat(candidateFile);
+      if (!stats.isDirectory()) {
```

Comment:
This should probably be `isFile()`. I don't think we want to accept other non-regular file entries (like device files).

Reply:
I wanted to also allow symbolic links as well and I felt like it was simpler and sufficient to just make sure it was not a directory. It is very unlikely that any other type of file will be created and if the user does manage to do that then they will see some other errors that will happen when we try to read the file.

Reply:
I believe that `fs.stat()` follows symlinks.